### DISCUSSION TITLE
想要使用apollo公共配置，java客户端appid必须是公共配置所在的项目id吗，我试的时候是这样的

### DISCUSSION THREAD

**Discussion by howHzz** [apolloconfig/apollo#3597] [Q&A]
- [ ] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [ ] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [ ] 我已经仔细检查过[FAQ](https://github.com/ctripcorp/apollo/wiki/%E9%83%A8%E7%BD%B2&%E5%BC%80%E5%8F%91%E9%81%87%E5%88%B0%E7%9A%84%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98)

**描述bug**

简洁明了地描述一下bug

**复现**

通过如下步骤可以复现：

1. 
2. 
3. 
4. 

**期望**

简介明了地描述你希望正常情况下应该发生什么

**截图**

如果可以，附上截图来描述你的问题

### 额外的细节和日志

- 版本：
- 错误日志
- 配置：
- 平台和操作系统

**Comment by howHzz** [comment]
版本1.8.0

**Comment by nobodyiam** [comment]
请详细描述可复现的步骤，比如基于 apollo demo 站点：http://106.54.227.205/
